recharges/models.py

```diff
@@ -4,14 +4,16 @@
 from django.db import models
 
 
-class DummyModel(models.Model):
+class RechargeModel(models.Model):
 
     """
     - Nice docstring please
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    product_code = models.CharField(max_length=20)
-    data = HStoreField(null=True, blank=True)
+    amount = models.DecimalField(max_digits=5, decima_places=2)
```

Comment:
decimal_places typo.
Does max_digits include the decimal_places? So the max recharge would be 999.99?  Or 99999.99?